README

```diff
@@ -16,7 +16,7 @@ Building ProArc
 ===============
 To build the release package use the following command:
 
-  mvn com.isomorphic:isc-maven-plugin:1.1.1:install -Dproduct=SMARTGWT -Dlicense=LGPL -DbuildNumber=6.0p -DbuildDate=2016-10-20
+  mvn com.isomorphic:isc-maven-plugin:1.1.1:install -Dproduct=SMARTGWT -Dlicense=LGPL -DbuildNumber=6.0p -DbuildDate=2016-10-23
```

Comment:
Z dlouhodobého hlediska to určitě lepší bude, ovšem tato oprava měla primárně zprovoznit build s co nejmenším zásahem do fungování aplikace (pokud by aktuálnější verze s-gwt něco rozbila).